Data/SessionService.cs

```diff
@@ -10,19 +10,82 @@ public sealed class SessionService
     public SessionService(IDbConnection dbConnection) =>
         _dbConnection = dbConnection;
 
-    public IReadOnlyList<Session> GetSessions() =>
-        _dbConnection.Query<Session, Author, Session>(
-            @"SELECT s.id AS sessionId, s.when, a.id AS authorId, a.name, a.surname 
-                FROM session s
-                JOIN author a
-                    ON s.chair_id = a.id",
-            (s, a) =>
+    public IReadOnlyList<Session> GetSessions()
+    {
+        var sessionDictionary = new Dictionary<int, Session>();
+        var lecturesDictionary = new Dictionary<int, Lecture>();
+        
+        var sessions = _dbConnection.Query<Session, Author, Lecture, Author, Paper, Author, Session>(
+            @"SELECT  s.id AS sessionId, s.when, 
+                a.id AS authorId, a.name, a.surname, 
+                l.id as lectureId, l.when, 
+                a2.id as authorId, a2.name, a2.surname, 
+                p.id as paperId, p.name, p.classification,
+                a3.id as authorId, a3.name, a3.surname
+                        FROM session s
+                        JOIN author a
+                            ON s.chair_id = a.id
+                        join lecture l 
```

Comment:
Tu powinien być `LEFT JOIN`, to zapytanie nigdy nie wypisze sesji, do której jeszcze nie dodaliśmy wykładu.